Cargo.toml

```diff
@@ -6,13 +6,13 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cfg-if = "1.0.0"
 hash32 = "0.3.1"
 
 [features]
 default = []
 use-std = []
 floats = []
+bitops = []
```

Comment:
out of curiosity: is there a reason to feature flag bitops --- why wouldn't i want them? is this something that some hardware platforms might not support (like floats?)

Reply:
So, this is maybe a point to think about - I think the original plan (?) was to feature gate a lot of this stuff to ensure it wouldn't be compiled in and affect binary size, so that you could always use the FULL_BUILTINS const.

I dunno how good the optimizer could be at seeing that if (for example) some float ops EXIST, but aren't used in a user-specified command list, that they would be correctly dead-code-eliminated.

Honestly, this might not be the best way to handle this, and I'm super not consistently feature gating it.

soooo... the answer is, I picked an arbitrary time to try to be consistent with an arbitrary previous decision, since bitops were "self contained" more or less. Probably needs to be reconsidered.